deviceadvisor/script/DATestRun.py

```diff
@@ -231,66 +242,75 @@ def sleep_with_backoff(base, max):
                     'thingArn': create_thing_response['thingArn'],
                 },
                 'parallelRun': True
-        })
+            })
 
         # get DA endpoint
         print("[Device Advisor] Info: Getting Device Advisor endpoint.")
         endpoint_response = deviceAdvisor.get_endpoint(
-            thingArn = create_thing_response['thingArn']
+            thingArn=create_thing_response['thingArn']
         )
         os.environ['DA_ENDPOINT'] = endpoint_response['endpoint']
 
         cycle_number = 0
         while True:
             cycle_number += 1
             if (cycle_number >= MAXIMUM_CYCLE_COUNT):
-                print(f"[Device Advisor] Error: {cycle_number} of cycles lasting {BACKOFF_BASE} to {BACKOFF_MAX} seconds have passed.")
-                raise Exception(f"ERROR - {cycle_number} of cycles lasting {BACKOFF_BASE} to {BACKOFF_MAX} seconds have passed.")
+                print(f"[Device Advisor] Error: {cycle_number} of cycles lasting {
```

Comment:
fix

Reply:
Though it's weird that autopep8 cannot process formatted strings properly